src/main/java/io/quarkus/bot/AnalyzeWorkflowRunResults.java

```diff
@@ -129,7 +129,7 @@ void analyzeWorkflowResults(@WorkflowRun.Completed GHEventPayload.WorkflowRun wo
                 artifactsAvailable,
                 checkRunOptional.orElse(null),
                 WorkflowConstants.MESSAGE_ID_ACTIVE);
-        if (!quarkusBotConfig.isDryRun()) {
+        if (!quarkusBotConfig.isDryRun() && (workflowRun.getConclusion() != Conclusion.CANCELLED)) {
```

Comment:
So you're doing all the work to generate the report and then you don't post it?

This check should be added at the top just after this one: https://github.com/quarkusio/quarkus-github-bot/pull/128/files#diff-934d6328af4061325fdd799d48620dee046978df99a47ef0f9e182218fdb30e7R69 .

Reply:
Does the line right under this one not post the report?